manifests/params.pp

```diff
@@ -0,0 +1,13 @@
+# == Class: puppetcrl_sync::params
+#
+# A set of default parameters
+#
+class puppetcrl_sync::params {
+
+  $crl_location = '/var/lib/puppet/ssl/crl.pem'
+  $crl_target   = '/var/lib/puppet/ssl/crl.pem'
```

Comment:
Are these defaults sane?  I mean, won't this just overwrite the same file if you just include the `puppetcrl_sync` class?